homeassistant/components/recorder/models.py

```diff
@@ -198,6 +210,36 @@ def to_native(self, validate_entity_id=True):
             return None
 
 
+class Statistics(Base):  # type: ignore
+    """Statistics."""
+
+    __table_args__ = {
+        "mysql_default_charset": "utf8mb4",
+        "mysql_collate": "utf8mb4_unicode_ci",
+    }
+    __tablename__ = TABLE_STATISTICS
+    id = Column(Integer, primary_key=True)
+    created = Column(DATETIME_TYPE, default=dt_util.utcnow)
+    source = Column(String(255), index=True)
+    statistic_id = Column(String(255), index=True)
```

Comment:
Should we have an index on `statistic_id, period, start` ?

Reply:
`period` column is gone now.
Index is there on `statistic_id` and `start`

Reply:
That's an index on a single column. You can have multi column indexes. SQLite will only use a single index when resolving a query, so a multi column index will be beneficial.

Reply:
Ok, multi column index for statistic_id + start is added in 66265ff

Reply:
We should drop the index on just `statistic_id` as it's already covered by `ix_statistics_statistic_id_start`. 
```suggestion
    statistic_id = Column(String(255), index=False)
```